ush/land/genYAML_output_letkfoi.yaml

```diff
@@ -20,7 +20,7 @@ driver:
   save posterior mean increment: 'true'
 geometry:
   akbk: Data/fv3files/akbk127.nc4
-  field metadata override: Data/fieldmetadata/gfs-land.yaml
+  field metadata override: Data/fieldmetadata/fv3jedi_fieldmetadata_restart.yaml
```

Comment:
Need change to: 
 field metadata override: $(DATA)/fv3jedi/fv3jedi_fieldmetadata_restart.yaml

Reply:
With the recent commit, this issue was resolved.